src/shared/components/challenge-listing/ChallengeCard/Status/index.jsx

```diff
@@ -374,7 +374,7 @@ class ChallengeStatus extends Component {
       return (
         <div styleName="avatar-container" key={winner.handle}>
           <UserAvatarTooltip user={userProfile}>
-            <LeaderboardAvatar member={winner} />
+            <LeaderboardAvatar member={winner} domain={config.URL.BASE} />
```

Comment:
@shubhendusaurabh Here, and in the other file, do we really need to pass in the domain via the props? I believe, as now we get it from config, we can just remove such prop and related typechecks, etc. Please correct the code (or correct me, if there are some use cases that still need custom domains.)